### PR TITLE
qa/tests - added for the suites with subset be able to use 'testing' …

### PR DESCRIPTION
**Pull request by yuriw**
…distro

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/machine_types/schedule_subset.sh [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-#command line => CEPH_BRANCH=<branch>; MACHINE_NAME=<machine_type>; SUITE_NAME=<suite>; ../schedule_subset.sh <day_of_week> $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL
+#command line => CEPH_BRANCH=<branch>; MACHINE_NAME=<machine_type>; SUITE_NAME=<suite>; ../schedule_subset.sh <day_of_week> $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL <$FILTER>
 
 # $1 - part (day of week)
 # $2 - branch name
 # $3 - machine name
 # $4 - suite name
 # $5 - email address
-# $6 - filter out (this arg is to be at the end of the command line for now)
+# $6 - distro
```
Comment (jdurgin) [inline qa/machine_types/schedule_subset.sh]: kernel

Comment (yuriw) [reply qa/machine_types/schedule_subset.sh]: done

**Comment by vasukulkarni** [comment]
The tests will start failing for CentOS and Ubuntu Distro Kernel?  Isn't that an unnecessary noise?

**Comment by yuriw** [comment]
test this please

**Comment by yuriw** [comment]
@vasukulkarni why will they fail? we are only making multimds use `testing` kernel

**Comment by yuriw** [comment]
tested manually and looked ok

**Pull request merged**